public/scss/_base.scss

```diff
@@ -141,6 +141,21 @@ li:not(:last-of-type) {
   @extend %page-container;
 }
 
+.stepper {
+  margin-bottom: 0;
+
+  p {
+  margin-top: $space-md;
+  margin-bottom: 5px;
+  color: $stepper-grey-light;
+  }
+
+}
+
+.stepperH1 {
+  margin-top: 0;
+}
```

Comment:
`+` is the "after" selector in CSS.

Unless you're doing something more elaborate, you can write this as:

```css
.stepper {
   ... CSS rules

  + h1 {
    margin-top: 0;
  }
}
```